src/project/manifest/pyproject.rs

```diff
@@ -71,16 +71,23 @@ impl From<PyProjectManifest> for ProjectManifest {
         manifest.project.name = Some(pyproject.name.clone());
 
         // Add python as dependency based on the project.requires_python property (if any)
-        let pythonspec = pyproject
+        let python_spec = pyproject
             .requires_python
             .clone()
             .map(VersionOrUrl::VersionSpecifier);
+
         let target = manifest.default_feature_mut().targets.default_mut();
-        target.add_dependency(
-            PackageName::from_str("python").unwrap(),
-            version_or_url_to_nameless_matchspec(&pythonspec).unwrap(),
-            SpecType::Run,
-        );
+        if !target.has_dependency("python", Some(SpecType::Run)) || python_spec.is_some() {
```

Comment:
We should not make the `requires-python` be more important then the pixi one. This won't let the user overwrite it if they want a different version from what the build package is allowed to use. e.g.
```toml
requires-python = `>=3.7` 
```
Will always give you the latest version but users might want to run their environment on `3.9` and then they can't overwrite that in the `tool.pixi` section.

Reply:
So we also don't want a warning, in that case?

Reply:
I don't think so, it will get annoying fast. Possibly an `tracing::info`

Reply:
Do we want to not do anything for other interpreters? Pypy, etc…